tests/test_backpressure.py

```diff
@@ -1,42 +1,77 @@
-# Copyright 2016-2021 XMOS LIMITED.
+# Copyright 2016-2022 XMOS LIMITED.
 # This Software is subject to the terms of the XMOS Public Licence: Version 1.
 import xmostest
 
-def do_test(sample_rate, num_channels, data_bits, receive_increment, send_increment, testlevel):
+
+def do_test(
+    sample_rate, num_channels, data_bits, receive_increment, send_increment, testlevel
+):
 
     resources = xmostest.request_resource("xsim")
 
     id_string = "{db}_{sr}_{nc}_{ri}_{si}".format(
```

Comment:
you could use python3 f strings here f"{val}.. etc. but not a big issue

Reply:
I've got an open issue to update testing anyway to use pytest rather than xmostest - will tackle along with that body of work!